src/sagemaker/jumpstart/model.py

```diff
@@ -347,10 +366,16 @@ def _create_sagemaker_model(self, *args, **kwargs):  # pylint: disable=unused-ar
                 container_def,
                 vpc_config=self.vpc_config,
                 enable_network_isolation=self.enable_network_isolation(),
-                tags=kwargs.get("tags"),
+                tags=tags,
             )
         else:
-            super(JumpStartModel, self)._create_sagemaker_model(*args, **kwargs)
+            super(JumpStartModel, self)._create_sagemaker_model(
+                instance_type=instance_type,
+                accelerator_type=accelerator_type,
+                tags=tags,
```

Comment:
Sanity check: will `tags` value end up being `None` here? Or will the values in `kwargs` cause a keyword argument name conflict?

Reply:
`tags` cannot be in `kwargs` since it's an explicit argument. `kwargs` will only contain new parameters should they get added in the future, to ensure it gets passed to the parent class.

Reply:
Just confirming that in the previous behavior, `kwargs.get("tags")` returned `None` b/c the `tags` key would not have existed

Reply:
+1, if just to make sure on above, if it does maybe we just append the tags from kwargs